addons/test_mail_full/static/tests/messaging_menu_patch_tests.js

```diff
@@ -12,7 +12,7 @@ QUnit.test("rating value displayed on the preview", async (assert) => {
     const partnerId = pyEnv["res.partner"].create({});
     const channelId = pyEnv["discuss.channel"].create({});
     const messageId = pyEnv["mail.message"].create([
-        { author_id: partnerId, model: "discuss.channel", res_id: channelId },
+        { author_id: partnerId, body: "non-empty", model: "discuss.channel", res_id: channelId },
```

Comment:
Test worked before change because the message was not considered empty thanks to `markup(false)` being truthy. Current implementation requires non-empty body on message to show rating in messaging menu, hence the change.